diy_asylum_frontend/src/components/FormContentComponent/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const inputMaker = ({ inputType, elementName, help }, helpTextSetter) => {
```

Comment:
I like this! This was what I was hoping to make when I briefly mentioned it in this slack message (https://diyasylum.slack.com/archives/CEPM8HDAL/p1546537263006700)

Why don't we just make these there own react components and use them explicitly tho?

It's elegant and abstracted logic as it stands right now, but it we should try and keep as close to the metal as possible in the FE.